src/main/java/io/codelex/java/advanced/test/Exercise3/Joiner.java

```diff
@@ -0,0 +1,26 @@
+package io.codelex.java.advanced.test.Exercise3;
+
+public class Joiner <T> {
```

Comment:
Pārlasi vēlreiz uzdevumu:
`Design the Joiner<T> class, which in the constructor will take a separator (string)`
Tavā gadījumā nevar padot citu separator.
Kā arī uzmet acī String join metodei.

Reply:
Jā, šajā gadījumā es pats sevi "izlaboju" un atstāju nepareizu variantu, jo no sākuma izveidoju mainīgo separator un tad izdzēsu to ārā, ātri pārlasot uzdevumu:

**e.g. for Integers and separator "-" it should return: 1-2-3-4 ...**

Tajā brīdī likās, ka vajag tikai "-"